tutorials/source-modeling/plot_fix_bem_in_blender.py

```diff
@@ -129,9 +138,68 @@
             op.join(bem_dir, 'inner_skull_orig.surf'))
 
 # Overwrite the original surface with the fixed version
+# In real study you should provide the correct metadata using ``volume_info=``
 mne.write_surface(op.join(bem_dir, 'inner_skull.surf'), coords, faces,
                   overwrite=True)
```

Comment:
Can you comment out this overwriting as well?

Reply:
`write_surface` and `overwrite` was used from the top to the bottom of the tutorial. I understand the danger of overwriting the files, should I modify the previous codes too? I will push a commit that way. If not I can just revert it.